Tests/Engine/DeadlockTestAssets/complex.psm1

```diff
@@ -0,0 +1,839 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+#region Private implementation functions
```

Comment:
is it worth providing a comment that says: line x seems to usually cause a deadlock when script analyzer has a go at it.